AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Itineraries/AddTeamMemberCommandHandlerAsync.cs

```diff
@@ -58,8 +56,10 @@ public AddTeamMemberCommandHandlerAsync(AllReadyContext context, IMediator media
                 _context.TaskSignups.Attach(taskSignup);
                 _context.Entry(taskSignup).Property(x => x.ItineraryId).IsModified = true;
                 await _context.SaveChangesAsync();
+
```

Comment:
`.ConfigureAwait(false)` for `await _context.SaveChangesAsync();` and `_mediator.SendAsync`